src/components/Camera.vue

```diff
@@ -117,6 +134,47 @@
       }
     }
 
+    private detachLettersGifs() {
+      this.$refs.letterGif.forEach( (letterGif: any) => {
+        letterGif.object3D.visible = false;
+      } );
+    }
+
+    private atachLettersGifs() {
+      this.$refs.letterGif.forEach( (letterGif: any) => {
+        letterGif.object3D.visible = true;
+      } );
+    }
+
+    private showWordGif(processedLetters: Marker[], wordGif: Image) {
+      this.detachLettersGifs();
+      const markerPositioned: Marker = WordsStats.markersMean(processedLetters);
+      this.$refs.wordGif.setAttribute('material', `shader:gif; src:url(${wordGif.url});`);
+      this.$refs.wordGif.object3D.position.set(
+        markerPositioned.position.x,
+        markerPositioned.position.y,
+        markerPositioned.position.z,
+      );
+
+      this.$refs.wordGif.object3D.scale.set(
+        markerPositioned.scale.x * 2.5,
+        markerPositioned.scale.y * 2.5,
+        markerPositioned.scale.z * 2.5,
+      );
+      this.$refs.wordGif.object3D.visible = true;
+      if (!this.$refs.wordGif.isPlaying) {
+        this.$refs.wordGif.play();
+      }
+    }
+
+    private detachWordGif() {
+      this.atachLettersGifs();
+      this.$refs.wordGif.object3D.visible = false;
+      if (this.$refs.wordGif.isPlaying) {
+        this.$refs.wordGif.pause();
+      }
+    }
```

Comment:
Vamos quebrar  umas linhas pra melhorar a visibilidade.

    private detachWordGif() {
      this.atachLettersGifs();

      this.$refs.wordGif.object3D.visible = false;

      if (this.$refs.wordGif.isPlaying) {
        this.$refs.wordGif.pause();
      }
    }